multidim-interop/src/generator.ts

```diff
@@ -79,14 +86,27 @@ export async function buildTestSpecs(versions: Array<Version>): Promise<Array<Co
             muxer: test.muxer,
             security: test.sec,
         })
-    )).concat(quicQueryResults.concat(quicV1QueryResults).concat(webtransportQueryResults).map((test): ComposeSpecification => buildSpec(containerImages, {
-        name: `${test.id1} x ${test.id2} (${test.transport})`,
-        dialerID: test.id1,
-        listenerID: test.id2,
-        transport: test.transport,
-        muxer: "quic",
-        security: "quic",
```

Comment:
This is a bit odd, I think leaving these blank would make the most sense. It requires the tests to parse the variables in steps but I don't see why that is necessarily a problem.

Reply:
Yeah agree Thomas, commented on https://github.com/libp2p/test-plans/pull/100#discussion_r1072194829

Reply:
I think this makes sense to me. We can make this change in a future PR so we do it across all impls